fe/fe-core/src/main/java/org/apache/doris/planner/PlanNode.java

```diff
@@ -870,4 +864,22 @@ public void convertToVectoriezd() {
             child.convertToVectoriezd();
         }
     }
+
+    // The slotIdSet could be null
+    public void initOutputSlotIds(Set<SlotId> slotIdSet, Analyzer analyzer) throws NotImplementedException {
```

Comment:
I think `slotIdSet` is a bit generic, what about `requiredSlotIds` ?
```suggestion
    public void initOutputSlotIds(Set<SlotId> requiredSlotIds, Analyzer analyzer) throws NotImplementedException {
```

Reply:
fixed